### DISCUSSION TITLE
Xiao ESP32S3 Sense and BH1750

### DISCUSSION THREAD

**Discussion by ali86x** [arendst/Tasmota#20661] [Q&A] [answered]
I sucessfully flashed my Xiao Sense Cam with Tasmota. Compiling with PlatformIO works great and the Camera is working fine.

Ist it possible to add support for BH1750 Sensors? They are not implemented in the Webcam Firmware. So maybe I need a Combination of Webcam and Sensor Firmware. 
Is that possible? 
Maybe someone can Help my with that. 

Thanks.

**Comment by Jason2866** [comment, marked as answer]
No, not possible to combine build variants. How to do is described in Tasmota documentation.
https://tasmota.github.io/docs/Compile-your-build/#customize-your-build

**Reply by ali86x** [reply]
Thanks for the answer. I used compiling to create a working Firmware für the Xiao Board. I Just need to know what to add now.
Is it enough to add "USE_I2C" and "USE_BH1750", or do I need to add Something else?

**Reply by Jason2866** [reply]
There will be no camera support. You have to add `USE_WEBCAM` to add the cam support.

**Reply by ali86x** [reply]
Sorry, maybe you so Not Unterstand because of my Bad english. 
I have already a running Version of tasmota-webcam with Support for the Xiao Sense Board. I "only" need to add support for the BH1750

**Reply by Jason2866** [reply]
You can NOT add to the Webcam Build support for the BH1750. The only builds where free configuration is possible are the build variants Tasmota and Tasmota32.

**Comment by ali86x** [comment]
Thanks for your hint.
IT works fine now. Using Tasmota32 and adding custom Environment for the Xiao Board including Support for bh1750.
Great Job!
